README.md

```diff
@@ -1,8 +1,45 @@
 # JanusGraph Ambari plugin
+JanusGraph is a scalable graph database optimized for storing and querying graphs containing hundreds of billions of vertices and edges distributed across a multi-machine cluster. JanusGraph is a transactional database that can support thousands of concurrent users executing complex graph traversals in real time.
+
+Instructions on how to add JanusGraph to Apache Ambari or Hortonworks Data Platform(HDP) as a service can be found below.
```

Comment:
- add links to https://ambari.apache.org and https://hortonworks.com/products/data-platforms/hdp/
- also specify what versions of Ambari and HDP this plugin supports

Reply:
Currently tested with 2.5 and 2.6. 2.7 breaks the install so I document the supported versions and hopefully be able to support the new version in the future without using a separate branch.